data/remnant/remnant jobs.txt

```diff
@@ -659,3 +659,37 @@ mission "Remnant: Rescue 5"
 	on visit
 		dialog
 			`You have returned, but there is still no word from the missing ship.`
+
+
+mission "Remnant: Disable 1"
+	job
+	repeat
+	name "Disable Damaged Kas'lor Ik 582"
+	description "A damaged Kas'lor Ik 582 is stuck in the Ember Waste sectors boarding Korath space. It has a large civilian population so we ask you to only disable it to avoid an atrocity. After this return to <planet> to receive your payment of <payment>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Wont Someone Please Think Of The Children: done"
+		random < 5
+	npc disable save
+		government "Korath"
+		personality heroic vindictive target marked staying
+		system 
+			near "Antevorta" 1
+		fleet
+			names "korath"
+			variant
+				"Kas'lor Ik 582 (Crippled)"
+		dialog `You have disabled the Kas'lor Ik 582, time to report back to <planet>.`
+	on visit
+		dialog "You've landed on <planet>, but the stuck Kas'lor Ik 582 has not been disabled yet. Hunt it down and disable it before returning."
+	on abort
+		"reputation: Remnant" += 250
+	on fail
+		"reputation: Remnant" -= 250
+		dialog
+			`You where unable to disable the Kas'lor Ik 582 without destroying it... The Remnant will not be pleased...`
+	on complete
+		"remnant: disable and save count" ++
+		payment 2000000
+		dialog "A Remnant military leader thanks you for disabling down the Kas'lor Ik 582 <npc>, and gives you the agreed-upon payment of <payment>."
```

Comment:
```suggestion
		dialog "A Remnant military prefect thanks you for disabling down the Kas'lor Ik 582 <npc>, and gives you the agreed-upon logistics adjustment of <payment>."
```

While basically being payment by another name, it is good to stick with the "logistics adjustment" framework to maintain consistency. (As in, they aren't *paying* the player, they are providing them with an adjustment to the amount of logistical support they can obtain, as required to deal with damage and expenses incurred in the process of the mission.)